src/Factory/IndexUrlFactoryInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SitemapPlugin\Factory;
+
+use SitemapPlugin\Model\IndexUrlInterface;
+
+interface IndexUrlFactoryInterface
+{
+    public function createNew(string $location): IndexUrlInterface;
```

Comment:
What about just `create`? Kind of implies that it's new.

Reply:
I think you have `createNew` in your factories, so I probably just copied that. And you probably did that because Sylius does it in their factories. Maybe we should just keep 'the Sylius way'?

Reply:
Sylius way for sure it is :)